src/components/PathwaysModal/PathwaysModal.tsx

```diff
@@ -0,0 +1,466 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import { useGetIsLoggedIn } from "hooks";
+import { routeNames } from "routes";
+import { BitzDropdown } from "../BitzShortcuts/BitzShortcuts";
+
+type PathwaysModalProps = {
+  showPathwaysModel: boolean;
+  handleHidePathwaysModel: any;
+};
+
+export const PathwaysModal: React.FC<PathwaysModalProps> = (props) => {
+  const { showPathwaysModel, handleHidePathwaysModel } = props;
+  const isLoggedInMvx = useGetIsLoggedIn();
+
+  const [pathwaysMenuItem, setPathwaysMenuItem] = useState<number>(1); // 1 datanft, 2 bitz, 3 liveliness, 4 passport, 5 bridge
```

Comment:
This should be an enum that has the correct naming instead of a comment explaining the "convention"

Reply:
Pathways is a WIP -- will be fixed in next release